metadata-db-app/src/cmr/metadata_db/data/memory_db.clj

```diff
@@ -68,6 +68,26 @@
          (ingest-events/concept-delete-event tombstone)))
       (concat concepts tombstones))))
 
+(defmethod after-save :service
+  [db concepts concept]
+  (if-not (:deleted concept)
+    concepts
+    (let [service-associations (concepts/find-latest-concepts
+                                db
+                                {:provider-id "CMR"}
+                                {:concept-type :service-association
+                                 :service-concept-id (:concept-id concept)})
+          tombstones (map (fn [ta] (-> ta
```

Comment:
name this var `tag-association` - it took me a minute to realize what `ta` stood for

Reply:
Refactored the anonymous function into a standalone function.